### PR TITLE
Permalinks

### PR DESCRIPTION
**Pull request by tupton**
Add Twitter and FB permalinks (links to the source) to each post in the timestamp.

### CODE REVIEW

File: facebook.coffee [hunk 1]
```diff
@@ -48,11 +48,13 @@ class Collector
                     created: new Date(item.created_time).getTime(),
                     name: item.from.name,
                     message: item.message,
+                    # TODO better way to get permalink?
+                    source: item.actions[0].link,
```
Comment (mbulman) [inline facebook.coffee:52]: how would there be a better way?

Comment (tupton) [inline facebook.coffee:52]: "actions[0]" is weak, I'm not sure if every item has "actions," and the only actions I've seen are called "Comment" and "Like," neither of which sound like they are to be used for permalinks.

Comment (mbulman) [inline facebook.coffee:52]: k.  we can leave the TODO.  as a note, we should start a file of facebook examples with descriptions of how they should be rendered, etc.  that's going to be one of the hardest parts.  first one to work on that can start the file.

Comment (tupton) [inline facebook.coffee:52]: Actually, it looks like those "permalinks" are predictably created from the item's id. I'll see if I can do something similar to what I did to create Twitter permalinks.

Comment (mbulman) [inline facebook.coffee:52]: Great timing.. just merged it ;)  I wonder if it'll reopen itself if you commit again

Comment (tupton) [inline facebook.coffee:52]: Doubtful. If not, I'll just open another one. No worries; I bet this happens a lot.

**Pull request merged**